### DISCUSSION TITLE
Pagination broken after v.3 upgrade?

### DISCUSSION THREAD

**Discussion by yabdab** [livewire/livewire#8183] [1. Help]
After upgrading Livewire  to v.3  I get this exception on pages WithPagination involved ...
```
PropertyNotFoundException
PHP 8.2.16
10.48.4
Property [$numberOfPaginatorsRendered] not found on component: [data-tables]
```

Any ideas on where I should begin  looking to fix this? Ran every cache clearing command I could think of, still not love. 😖

**Comment by yabdab** [comment]
Looks like it was due to a custom pagination view, that worked fine in v2, but now does not like the `numberOfPaginatorsRendered` property. Removing that fixed things.

**Reply by joshhanley** [reply]
@yabdab yep always a good idea to republish the pagination views after a major update and make any required changes to any custom ones you have.
